### ISSUE TITLE
[Feature Request] Add Power button/suspend

### ISSUE THREAD

**Issue by VandaLHJ** [micheleg/dash-to-dock#515]
Add option to have power button that let you suspend/power off/sleep that is pined down to bottom of panel. Possibly let users put other icons in that area (like trash can for example).

**Comment by micheleg** [comment]
I don't think this feature makes a lot of sense. The way I see it this is already well served by the top panel. If you want to integrate all functionalities in one place you might be interested in this other extension https://github.com/jderose9/dash-to-panel.

Regardin additonal icons on the dock, this has been discussed already in the past. Ha a look at the other bug reports.
